frontend/packages/data-portal/app/components/Filters/IdFilter.tsx

```diff
@@ -0,0 +1,40 @@
+import { useMemo } from 'react'
+
+import { QueryParams } from 'app/constants/query'
+
+import { RegexFilter } from './RegexFilter'
+
+export function IdFilter({
+  id,
+  label,
+  title,
+  queryParam,
+  prefix,
+}: {
+  id: string
+  label: string
+  title: string
+  queryParam: QueryParams
+  prefix?: string
+}) {
+  const validatingRegex = useMemo(
+    () => (prefix ? RegExp(`^(?:${prefix}-)?(\\d+)$`, 'i') : /^\d+$/),
```

Comment:
since the filter isn't using the capture groups from the regex anymore, is it possible to just remove `?:`?